src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsSynchronisationService.kt

```diff
@@ -30,15 +41,19 @@ class AlertsSynchronisationService(
         // TODO - probably update alert??
       } else {
         dpsApiService.createAlert(nomisAlert.toDPsAlert()).run {
-          // TODO - deal wit error scenario
-          mappingApiService.createMapping(
-            AlertMappingDto(
-              dpsAlertId = this.alertUuid.toString(),
-              nomisBookingId = nomisAlert.bookingId,
-              nomisAlertSequence = nomisAlert.alertSequence,
-              mappingType = DPS_CREATED,
-            ),
-          )
+          tryToCreateMapping(
+            nomisAlert = nomisAlert,
+            dpsAlert = this,
+            telemetry = telemetry,
+          ).also { mappingCreateResult ->
+            val additionalTelemetry =
+              if (mappingCreateResult == MAPPING_CREATED) mapOf() else mapOf("mapping" to "initial-failure")
+
+            telemetryClient.trackEvent(
+              "alert-synchronisation-created-synchronisation-success",
```

Comment:
did you want synchronisation twice here

Reply:
I want it 22 times. Thanks, I'll correct in next PR